benches/coprocessor_executors.rs

```diff
@@ -35,6 +35,8 @@ fn bench_table_scan_next(
     ranges: &[KeyRange],
     store: &Store<RocksEngine>,
 ) {
+    use tikv::coprocessor::dag::executor::TableScanExecutor;
```

Comment:
What's the benefit of putting it here instead of the header?

Reply:
To avoid conflict between `TableScanExecutor` and `IndexScanExecutor`? @breeswish

Reply:
It's only a personal taste. For me, `TableScanExecutor` doesn't seems to be a useful import entry for future additions and it is only for part of the file. So I put it in the partial scope instead of global scope. If in future we have a standalone `.rs` file specifically for TableScanExecutor, I would put it in global scope again.